src/sources/metadatasourcetaglib.cpp

```diff
@@ -745,9 +749,11 @@ class SafelyWritableFile final {
 std::pair<MetadataSource::ExportResult, QDateTime>
 MetadataSourceTagLib::exportTrackMetadata(
         const TrackMetadata& trackMetadata) const {
-    kLogger.debug() << "Exporting track metadata"
-            << "into file" << m_fileName
-            << "with type" << m_fileType;
+    if (kLogger.debugEnabled()) {
+        kLogger.debug() << "Exporting track metadata"
```

Comment:
I am not sure if the file name is logged somewhere else as well. If not this one is important to identify a faulty file that might be the cause for a taglib crash.